model.py

```diff
@@ -10851,6 +10852,8 @@ def numericrange_to_tuple(r):
 
 def tuple_to_numericrange(t):
     """Helper method to convert a tuple to an inclusive NumericRange."""
+    if not t:
+        return None
```

Comment:
This allows `target_age` to return None if it wants a subject to not be considered for target age at all.